api.txt

```diff
@@ -0,0 +1,22 @@
+AC1:   POST /companies/{id}  requestbody{name}   =>201,400
```

Comment:
POST /companies  no id information in uri

Reply:
agreed